assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -31,7 +31,7 @@ import { getPastedCoordinates } from '../../utils/copyPaste';
 const MOVE_COARSE_STEP = 10;
 
 /**
- * @param {{current: Node}} ref
+ * @param {MutableRefObject<HTMLElement | null>} ref
```

Comment:
little tighter type here. nice to see doc strings accepts TS types

Reply:
This is great. We should do a pass and clean it up everywhere.